config/samples/core_v1alpha1_humiofeatureflag.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: core.humio.com/v1alpha1
+kind: HumioFeatureFlag
+metadata:
+  name: humiofeatureflag-example
+spec:
+  managedClusterName: example-humiocluster
+#  externalClusterName: example-humiocluster
+  name: example-featureflag
```

Maybe we should add an example of a real flag rather than this non-existent flag. I suggest we update it to this:
```suggestion
  name: MultiClusterSearch
```

---

Good point!